samples/bot-task-module/nodejs/pages/CustomForm.html

```diff
@@ -9,7 +9,7 @@
     <script src="https://statics.teams.cdn.office.net/sdk/v1.5.2/js/MicrosoftTeams.min.js" asp-append-version="true"></script>
```

Comment:
Still unexpected changes for bot-task-module? I think there would be zero change for bot-task-module.